tests/core/test_memory.py

```diff
@@ -0,0 +1,161 @@
+import pytest
+import torch
+import torch.nn as nn
+
+from benchmarks.parity_modules import ParityModuleRNN
```

Comment:
pls do not import anything from `benchmarks`, it is meant to completely independent...
if you need this model do it another way around - move it to `tests.base` and we can import it in benchmarks

it may also cause a problem with cyclic imports as benchmarks already use `tests.base.utils`

Reply:
why is benchmarks anyway not a subfolder of test?
let's just move it entirely? I will have to make a separate PR I guess.

Reply:
because we do not want to have benchmarks always, but maybe it could be inside tests
cc: @williamFalcon

Reply:
i moved the modules to the test folder, but left the benchmark where it is